developer-docs-site/docs/guides/building-your-own-wallet.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Building Your Own Wallet"
+slug: "building-your-own-wallet"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building Your Own Wallet
+
+There are already a few wallets building on Aptos. To allow for easy wallet interoperability we need to do the same things across wallets:
+1. Mnemonics
+2. Dapp API
+3. Handling key rotation 
+
+## Mnemonics
+While [Petra wallet](../install-petra-wallet.md) recommends 1 mnemonic <-> 1 account, we recognize that some wallets may want to support 1 mnemonic <-> n accounts coming from other chains. To support both of these use cases we are using a [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) derive path for mnemonics to accounts.
+
+### Creating an Aptos Account
+1. Generate mnemonic using something like BIP39
+2. Get master seed from that mnemonic using BIP39
+3. Use BIP44 derive path to retrieve an account address (e.g. `m/44'/637'/0'/0'/0'`)
+    - See Aptos' [typescript sdk's implementation for the derive path](https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/aptos_account.ts#L48-L69)
+    - In the case of Petra, we will always use the path `m/44'/637'/0'/0'/0'` since we have 1 mnemonic <-> 1 account
+
+
+```
```

Comment:
might be useful to add the right parser here.

Reply:
I kinda want people to use the sdk version, I included the code to give context but don't want to get into specifics